### PR TITLE
Improve thumbnail scaling

### PR DESCRIPTION
**Pull request by dmitryduev**
This PR accompanies [this K PR](https://github.com/dmitryduev/kowalski/pull/22) and improves ZTF alert thumbnail scaling.

### CODE REVIEW

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 1]
```diff
@@ -490,28 +477,66 @@ def post(self, objectId):
...
+                        "magerr": df.loc[pid_mask, "sigmapsf"].tolist(),
+                        "limiting_mag": df.loc[pid_mask, "diffmaglim"].tolist(),
+                        "magsys": df.loc[pid_mask, "magsys"].tolist(),
+                        "filter": df.loc[pid_mask, "ztf_filter"].tolist(),
+                        "ra": df.loc[pid_mask, "ra"].tolist(),
+                        "dec": df.loc[pid_mask, "dec"].tolist(),
+                    }
+
+                    if len(photometry.get('mag', ())) > 0:
+                        photometry_handler = PhotometryHandler(request=self.request, application=self.application)
```
Comment (stefanv) [inline extensions/skyportal/skyportal/handlers/api/alert.py:531]: @dannygoldstein You've refactored similar code recently; do we have a better way than this of calling sub-handlers?  I thought we settled on a sign-in-via-generic-token approach.

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 2]
```diff
@@ -808,15 +841,30 @@ async def get(self, objectId: str = None):
...
+            elif interval.lower() == "min_max":
+                interval = MinMaxInterval()
+            elif interval.lower() == "zscale":
+                interval = ZScaleInterval(
+                    nsamples=600,
+                    contrast=0.045,
+                    krej=2.5
+                )
+
+            if (stretch is None) or (stretch.lower() not in ["linear", "log", "asinh", "sqrt"]):
```
Comment (stefanv) [inline extensions/skyportal/skyportal/handlers/api/alert.py:854]: Call `stretch.lower()` once, and use a dictionary lookup?

**Review by stefanv — COMMENTED** [review]
Needs some input from @dannygoldstein on one question.

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 1 above)
Comment (dmitryduev) [reply extensions/skyportal/skyportal/handlers/api/alert.py]: Sorry, my origin was outdated, fixed that. As for the approach, yes, we settled on it, but I haven't yet implemented it :)

**Review by stefanv — APPROVED** [review]

**Pull request merged**